guides/common/modules/proc_configuring-selinux-to-permit-access-to-repos-on-custom-ports.adoc

```diff
@@ -0,0 +1,25 @@
+[id="configuring-selinux-to-permit-access-to-repositories-on-custom-ports_{context}"]
+= Configuring SELinux to Permit Access to Repositories on Custom Ports
+
+ifdef::satellite,foreman-el,katello[]
+SELinux permits access of {Project} to repositories only on specific ports.
+By default, connecting to web servers running on the following ports is permitted: 80, 81, 443, 488, 8008, 8009, 8443, and 9000.
+
+endif::[]
+
+
+.Procedure
+. On {Project}, to verify the ports that are permitted by SELinux for repositories, enter a command as follows:
++
+[options="nowrap",subs="+quotes"]
+----
+# semanage port -l | grep ^http_port_t
+http_port_t     tcp      80, 81, 443, 488, 8008, 8009, 8443, 9000
+----
+
+. To configure SELinux to permit a port for repositories, for example 10011, enter a command as follows:
++
+[options="nowrap",subs="+quotes"]
+----
+# semanage port -a -t http_port_t -p tcp 10011
+----
```

Comment:
If the web proxy is running on a non-standard proxy port that is defined for some other purpose (8082, labelled us_cli_port_t):
$ semanage port -m -t http_port_t -p tcp 8082
(This means 8082 now has both labels on it)

I'm updating the upstream pulpcore-selinux docs with this too:

https://github.com/pulp/pulpcore-selinux/pull/42